Task6/css/header.css

```diff
@@ -0,0 +1,98 @@
+body {
+    width: 1200px;
+    margin: 0 auto;
+}
```

Comment:
What do styles for `body` do in `header.css` file?

Reply:
Fixed.